### PR TITLE
Handwritten Digit classification

### PR DESCRIPTION
**Pull request by praggyaverma**
Issue no. #4 
Obtained Accuracy:
image

CNN (Sequential) was used in model building. For training, the epoch was set to 5 with batch_size = 200
(I tried tuning the hyperparameter and obtained the highest accuracy in batch_size = 200)

### CODE REVIEW

**Comment by Nitya-Pasrija** [comment]
Please do not create multiple commits in a single PR

**Comment by praggyaverma** [comment]
> Please do not create multiple commits in a single PR

oh no! so sorry, I didn't know this before. I'll take care next time
